src/utils/log.js

```diff
@@ -0,0 +1,11 @@
+export const note = params => console.log(params);
+
+export const info = params => console.info(params);
```

Comment:
Do you plan to use all of these?  I have only seen two use cases in the code so far:  Logging exceptions and ephemeral debugging logs.  I suggest making only those two for now

Reply:
Since logging errors is the major use case, I suggest looking at https://github.com/klahnakoski/firefox-code-coverage-frontend/issues/25 to ensure the API can handle chaining (even if the implementation does nothing).